h/registerProduct.h

```diff
@@ -0,0 +1,37 @@
+
+
+const char *MODULE_REGISTER_USAGE = "IFAUSAGE";
+
+char productReg[COMMON_PATH_MAX];
+char productPID[COMMON_PATH_MAX];
+char productVer[COMMON_PATH_MAX];
+char productOwner[COMMON_PATH_MAX];
+char productName[COMMON_PATH_MAX];
+char productFeature[COMMON_PATH_MAX];
+
+typedef struct IFAARGS {
```

Comment:
This is only used by `registerProduct.c`, no need to expose it via a header.

Reply:
moved to registerProduct.c